explorer/e2e/testFunctions.ts

```diff
@@ -42,7 +49,10 @@ export async function testTab(page: Page, tab: TabDescription): Promise<void> {
   }
 }
 
-export async function testSort(page: Page, tab: TabDescription): Promise<void> {
+export async function testSortAzure(
```

Comment:
`Azure` should be replaced with `Azul`, and then I think this will be good to go!